scripts/vulkaninfo_generator.py

```diff
@@ -610,7 +610,7 @@ def PrintChainBuilders(listName, structures, all_structures):
                 # We need to artificially increase it just so the driver doesn't write 'out of bounds' and cause
                 # difficult to debug crashes. This bug comes from the in-development version of the extension having
                 # a larger size than the final version, so older drivers try to writ to members which don't exist.
-                if s.sTypeName == "VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_SHADER_INTEGER_DOT_PRODUCT_FEATURES_KHR":
+                if s.sTypeName == "VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_SHADER_INTEGER_DOT_PRODUCT_FEATURES":
```

Comment:
Should there be a check for both of these?

Reply:
No, because the xml uses only the `SHADER_INTEGER_DOT_PRODUCT_FEATURES` and the KHR version will only appear as an alias, not a distinct structure type.